stories/SkipLink.stories.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+  import {Meta, Story} from '@storybook/addon-svelte-csf';
+</script>
+
+<Meta
+  title="Components/Skip Link"
+  args={{
+    label: 'Hopp til hovedinnholdet',
+  }}
+/>
+
+<Story name="Hidden focusable link" let:label>
+  <a href="" class="visually-hidden-focusable">{label}</a>
```

Comment:
You actually get the wrong semantics if `href` doesn't have a value. It's a bit weird.

```suggestion
  <a href="#" class="visually-hidden-focusable">{label}</a>
```